vivisect/parsers/pe.py

```diff
@@ -38,7 +38,7 @@ def align(v, alignment):
     if remainder == 0:
         return v
     else:
-        return v + remainder
+        return v + (alignment - remainder)
```

Comment:
nice catch!

also, uh-oh, i copied this from my commits to two other projects recently 🤦🏼 silly arithmetic